app/src/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -265,7 +265,7 @@ private static FormDef deserializeFormDef(Context context, File formDef) {
             DataInputStream dis = new DataInputStream(new BufferedInputStream(fis));
 
             // read serialized formdef into new formdef
-            fd.readExternal(dis, ApkUtils.getPrototypeFactory(context));
+            fd.readExternal(dis, DbUtil.getPrototypeFactory(context));
```

Comment:
Note: this change means that we are now using an `AndroidPrototypeFactory` when loading forms, not a vanilla `PrototypeFactory`. I assume that is what we want, but am not positive.